tracing-subscriber/src/fmt/time.rs

```diff
@@ -219,8 +219,8 @@ where
         write!(writer, "{}", style.suffix())?;
     } else {
         timer.format_time(writer)?;
-        write!(writer, " ")?;
     }
+    write!(writer, " ")?;
```

Comment:
TIOLI: Maybe this should be 
```suggestion
    writer.write_char(' ')?;
```
rather than the macro?